cmd/sovereignnode/incomingHeader/incomingHeaderProcessor_test.go

```diff
@@ -441,20 +401,18 @@ func TestIncomingHeaderHandler_AddHeaderErrorCases(t *testing.T) {
 
 		handler, _ := NewIncomingHeaderProcessor(args)
 		err := handler.AddHeader([]byte("hash"), incomingHeader)
-		require.ErrorContains(t, err, "cannot deserialize event data")
+		require.ErrorContains(t, err, errCannotDeserializeEventData.Error())
 	})
 
 	t.Run("cannot create token data, should return error", func(t *testing.T) {
 		t.Parallel()
 
-		args := createArgs()
+		errCannotDeserializeTokenData := fmt.Errorf("cannot deserialize token data")
```

Comment:
Check below the error is this variable, not that it contains the text

Reply:
the `err` gives more info than `errCannotDeserializeTokenData` so I need contains

Reply:
Please change require below to

```go
		require.ErrorContains(t, err, errCannotDeserializeTokenData.Error())
```